### DISCUSSION TITLE
vulnerability CVE-2023-25193

### DISCUSSION THREAD

**Discussion by sabahsawor** [harfbuzz/harfbuzz#4463] [General] [answered]
Hi All,

My intrusion detection showing vulnerability CVE-2023-25193 affects libharfbuzz0b, I have version 2.7.4-1 installed on my Debian 11.
Can someone guide me on how to upgrade the version of libharfbuzz0b to late version or how to patch the old version?

**Comment by behdad** [comment, marked as answer]
I suggest you try applying this patch:
https://github.com/harfbuzz/harfbuzz/commit/85be877925ddbf34f74a1229f3ca1716bb6170dc

**Reply by sabahsawor** [reply]
Thank you for your answer.
How to apply this patch to my already-installed libharfbuzz0b

**Reply by behdad** [reply]
You would need to rebuild the Debian package. I don't know how to do that.

That said, this CVE is very minor issue that you would be fine without fixing.

**Reply by sabahsawor** [reply]
thank you
